frontend/src/pages/ExperimentList.tsx

```diff
@@ -177,6 +182,20 @@ export class ExperimentList extends Page<{ namespace?: string }, ExperimentListS
     let response: ApiListExperimentsResponse;
     let displayExperiments: DisplayExperiment[];
     try {
+      // This ExperimentList page is used as the "All experiments" tab
+      // inside ExperimentAndRuns. Here we only list unarchived experiments.
+      // Archived experiments are listed in "Archive" page.
+      const filter = JSON.parse(
```

Comment:
Discussion: Is it possible to reuse this component?
For example, this component can show unarchived experiments by default, but it has an optional prop that can show archived experiments instead.

You can notice that `RunList` component is also reused similarly in multiple places, both as a page or as part of a page.

I'm asking this, because `ExperimentList` is already a disaster to maintain, ... it would be twice the disaster if we copy it one more time.